SilMock/tests/Google/Service/DirectoryTest.php

```diff
@@ -399,6 +420,8 @@ public function testUsersUpdate_ById()
             "password" => "testP4ss",
             "primaryEmail" => "user_test4@sil.org",
             "suspended" => false,
+            "isEnforcedIn2Sv" => true,
+            "isEnrolledIn2Sv" => true,
```

Comment:
Are there any tests where you set either or both of these to `false`?

Reply:
No, though I had considered toggling some. Perhaps set it to true/false, because before it was null/null when not set? :)